src/lsst/cmservice/client/wrappers.py

```diff
@@ -247,23 +247,64 @@ def get_object_by_fullname_function(
     Returns
     -------
     the_function: Callable
-        Function that returns a single row from a table by name
+        Function that returns a single row from a table by fullname
     """
 
-    def get_obj_by_fullname(
+    def get_row_by_fullname(
         obj: CMClient,
         fullname: str,
     ) -> response_model_class:
         params = models.FullnameQuery(
             fullname=fullname,
         )
         results = obj.client.get(f"{query}", params=params.model_dump()).json()
+        if "detail" in results:
```

Comment:
Whut? Why? <stratches head>

Reply:
Cuz that's the easiest way to tell it bombed...  If you have a better way, I'm happy to use it.